src/symbols/dep_analyzer.rs

```diff
@@ -342,6 +343,18 @@ impl Visit for DepsFiller {
     }
   }
 
+  fn visit_arrow_expr(&mut self, n: &ArrowExpr) {
+    if let Some(type_params) = &n.type_params {
+      self.visit_ts_type_param_decl(type_params);
+    }
+    for param in &n.params {
+      self.visit_pat(param);
+    }
+    if let Some(return_type) = &n.return_type {
+      self.visit_ts_type_ann(return_type);
+    }
+  }
```

Comment:
We need to visit the body expression if it is not a block, and there is no return type. Test case:

```ts
type y = string;
export const x = () => {} as y;
```

Reply:
Good catch! Added a test under specs.